solr/bin/solr.in.sh

```diff
@@ -280,4 +280,4 @@
 # SOLR_OPTS="$SOLR_OPTS -Dlog4j2.formatMsgNoLookups=true"
 
 # The bundled plugins in the "modules" folder can easily be enabled as a comma-separated list in SOLR_MODULES variable
-# SOLR_MODULES=extraction,ltr
```

Comment:
Why is this one special compared to say SOLR_OPTS or others?

Reply:
The `bin/solr` script picks up `SOLR_OPTS` and other env.vars and pass them in as java sysprops on the cmdline. 

With `SOLR_MODULES` the Java code (https://github.com/apache/solr/blob/main/solr/core/src/java/org/apache/solr/util/ModuleUtils.java#L59:L71) parses both sys.prop and env.var. But it obviouly only works if the var is exported. It's a pity we need to touch the `bin/solr` and `bin/solr.cmd` scripts for every variable we want to add support for.

Reply:
This change is also required in solr.in.cmd

Reply:
I think https://github.com/apache/solr/pull/1007 is a better approach

Reply:
Agree, it's safer to follow the beaten track of parsing the var in bin/solr. Closing this.